website/src/pages/StatusPage/StatusPage.tsx

```diff
@@ -21,6 +21,15 @@ import useApiCall from "../../utils/Hook";
 import Hero from "./components/Hero";
 import ExecutionQueue from "./components/PreviousExecutions";
 import PreviousExecutions from "./components/PreviousExecutions";
+import datas from "./data.json";
```

Comment:
This should be removed, I am guessing it is the json that mocks the data.

Reply:
Done @frouioui